### ISSUE TITLE
[REQ] Add an option to start archiving only on click of a button

### ISSUE THREAD

**Issue by navinthomas** [aonez/Keka#1103]
### Is your feature request related to a problem? Please describe

The current UX for compressing multiple files in Keka is not very convenient for me personally. If I have to compress 5 different files into one archive, I have to drag them into Keka at the same time. This is easy and simple when the files are in the same folder and when they are close together. But, it can be quite difficult when they are many files, and when the files are in different folders. It's difficult to keep all of them selected at the same time before dragging into Keka.

### Describe the solution you'd like

Just like in typical archivers on Windows, it would be nice if there is an option within Keka to let users keep on adding files to be archived one by one, and then when all the files have been added, click an 'Archive/Compress/Start/OK' button to trigger the archive creation, instead of Keka automatically starting the archive creation after the first time the user drags something into the app.

Basically, an option to allow users to temporarily "hold" files in the app, before they want to actually start archive creation. This would allow a user to keep on adding files in multiple drag and drop operations until they have all the files necessary in the app, and then click a button to archive them.

### Describe alternatives you've considered

Alternatives that I have tried so far is to use any drag and drop shelf app. I drag the desired files to the shelf one step at a time and then drag them all together into Keka.

### Additional context
N/A
